README.md

```diff
@@ -122,6 +122,7 @@ This suite is being used by:
 * [networknt/json-schema-validator](https://github.com/networknt/json-schema-validator)
 * [Justify](https://github.com/leadpony/justify)
 * [Snow](https://github.com/ssilverman/snowy-json)
+* [jsonschemafriend](https://github.com/jimblackler/jsonschematypes/tree/master/library)
```

Comment:
Awesome, congrats! Could you snip the tree/master off the end so it just lands on your project page?

Otherwise will merge!

Reply:
Thank you Julian. It was done like that because the library was built as a submodule of the project it was written for (jsonschematypes).

I can see how that isn't ideal so I've moved it to its own repo.

Reply:
Ah sorry I didn't notice that -- that would have been fine too, apologies for not noticing, but yeah all good.